pkg/manifest/tests/results/fileTree_filtering.yaml

```diff
@@ -0,0 +1,9 @@
+
+- file: two.md
+  type: file
+  source: https://github.com/gardener/docforge/blob/master/contents/blogs/2024/two.md
```

Comment:
Can we also add a `.yaml` file to `contents/blogs/2024`? Because now we are only loading md files